src/Infrastructure.Dapper/Repositories/CollectionRepository.cs

```diff
@@ -309,6 +312,212 @@ public async Task<ICollection<CollectionAccessSelection>> GetManyUsersByIdAsync(
         }
     }
 
+    public async Task CreateDefaultCollectionsAsync(Guid organizationId, IEnumerable<Guid> orgUserIds, string defaultCollectionName)
```

Comment:
@eliykat for the list of `orgUserIds` should we pass in the requirement objects with the OrgUserId, just the `orgUserIds` like it is here

Reply:
Just saw this - the list of OrgUserIds is fine 👍